src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < 3; i++) {

```

Comment:
The loop for generating random figures has a hardcoded limit (3). Use a constant or a variable to set the limit, making it easier to adjust later.